interop-testing/src/main/java/io/grpc/testing/integration/TestServiceImpl.java

```diff
@@ -186,13 +195,22 @@ public void onError(Throwable cause) {
     return new StreamObserver<StreamingOutputCallRequest>() {
       @Override
```

Comment:
Why is the boolean in the `StreamObserver` preferred? I can do this, but it still requires some kind of check in `ResponseDispatcher#dispatchChunk()`, see below...